### PR TITLE
Users new and create

### PR DESCRIPTION
**Pull request by Jchew1234**
Lots of changes

### CODE REVIEW

**Review by a-jennings — APPROVED** [review]
lgtm

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 class ApplicationController < ActionController::Base
+
```
Comment (izumilink95) [inline app/controllers/application_controller.rb:2]: remove line pls it stresses me and rubocop out

File: config/routes.rb [hunk 2]
```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
+
```
Comment (izumilink95) [inline config/routes.rb:2]: remove line pls it stresses me and rubocop out

**Review by XR89 — APPROVED** [review]

**Review by izumilink95 — APPROVED** [review]

**Comment by izumilink95** [comment]
39

**Pull request merged**